src/main/java/com/feelmycode/parabole/service/EventService.java

```diff
@@ -37,32 +36,33 @@ public class EventService {
     private final ProductRepository productRepository;
 
     /**
-     *  이벤트 생성
+     * 이벤트 생성
      */
     @Transactional
     public Long createEvent(EventCreateRequestDto eventDto) {
 
         // 엔티티 조회
-        // TODO : Error Exception 수정
-        //Seller seller = sellerRepository.findById(eventDto.getSellerId()).orElseThrow(() -> new IdNotFoundException("해당하는 ID의 판매자가 없습니다."));
+        //Seller seller = sellerRepository.findById(eventDto.getSellerId()).orElseThrow(() -> new ParaboleException(HttpStatus.NOT_FOUND, "해당하는 ID의 판매자가 없습니다")));
         Long sellerId = eventDto.getSellerId();
 
         List<Long> productIds = eventDto.getEventPrizeCreateRequestDtos().getProductIds();
-        List<Long> couponIds  = eventDto.getEventPrizeCreateRequestDtos().getCouponIds();
+        List<Long> couponIds = eventDto.getEventPrizeCreateRequestDtos().getCouponIds();
 
         // 이벤트-경품정보 생성
         List<EventPrize> eventPrizeList = new ArrayList<>();
 
         if (CollectionUtils.isEmpty(productIds)) {
             for (Long productId : productIds) {
-                Product product = productRepository.findById(productId).orElseThrow(() -> new IdNotFoundException("해당하는 ID의 상품이 없습니다."));
-                EventPrize productPrize = new EventPrize("PRODUCT", eventDto.getEventPrizeCreateRequestDtos().getStock(), product);
+                Product product = productRepository.findById(productId).orElseThrow(() ->
+                    new ParaboleException(HttpStatus.NOT_FOUND, "해당하는 ID의 상품이 없습니다."));
```

Comment:
404 Not found는 페이지를 찾을 수 없을때 보내는 상태입니당! 바꿔주세요~
```suggestion
                    new ParaboleException(HttpStatus.BAD_REQUEST, "해당하는 ID의 상품이 없습니다."));
```

Reply:
리소스 없는 경우에도 404 NOT FOUND 사용가능한 걸로 협의완료